website/src/models/group.js

```diff
@@ -210,23 +210,16 @@ schema.methods.startQuest = async function startQuest (user) {
     user.markModified('party.quest');
   }
 
-  // Remove the quest from the quest leader items (if he's the current user)
+  // Remove the quest from the quest leader items (if they are the current user)
```

Comment:
No, absolutely. Just wanted to be sure of what it did. Also the number of updates is the same so the difference should be minimal. For me it looks good if you want to merge